### ISSUE TITLE
Support writing text fields which are longer than 255 characters.

### ISSUE THREAD

**Issue by scw** [R-ArcGIS/r-bridge#14]
As reported by Andrew Gilbert, currently if writing a text field, writing will fail if any of the values exceed 255 characters in length. Here's a quick example, with a String field that'll fit in 1024 characters:

`data_type_test.csv`
```csv
OBJECTID,OBJECTID_1,Shape_Length,StringField,Number,Boolean,LongString
,1,457754.519544604641851,,,TRUE,"This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string, This is a very long string."
```
Now, try to write this input using `arcgisbinding`:
```r
library(arcgisbinding)
arc.check_product()

# you'll need to set a location with setwd(), here I use a location with an
# existing `r_test.gdb` file.
# setwd()

d <- arc.open("data_type_test.csv")
dr <- arc.select(d)
arc.write("r_test.gdb/long_string_write", dr)
# fails with:
# Error in .call_proxy("arc_export2dataset", path, data, coords, shape_info) : 
#  The row contains a bad value. [LongString]
```
I discussed this with @dpavlushko, he said it's because of how we create the output. We basically do this in two steps, initializing an output dataset, and then later writing it. At initialization, we don't know the structure of the data, so it is initialized to the default length for a text field currently.

**Comment by matthewpaulking** [comment]
Just wanted to chime in and say that this would be extremely helpful to me in my work to increase the character limit. I'm using FileGDBs every day, and the ability to write out to .gdb inside of R is fantastic, but I do end up having to truncate some of my text fields. Appreciate all the work on this package!

**Comment by dpavlushko** [comment]
Fixed in [1.0.1.229](https://github.com/R-ArcGIS/r-bridge/releases/tag/v1.0.1.229-beta)
